js/functions.js

```diff
@@ -0,0 +1,26 @@
+/*Функция для проверки длины строки.
+Она принимает строку, которую нужно проверить, и максимальную длину
+и возвращает true, если строка меньше или равна указанной длине,
+и false, если строка длиннее.*/
+
+const lengthOfString = function (string, maxLength) {
+  return string.length <= maxLength;
+};
```

Comment:
У вас есть дополнительный критерий о том, чтобы по умалчанию юзать стрелки?
```suggestion
const lengthOfString = (string, maxLength) => string.length <= maxLength;
```